DigraphyApi/Controllers/CollectionController.cs

```diff
@@ -0,0 +1,21 @@
+using DigraphyApi.Dto;
+using DigraphyApi.Models;
+using DigraphyApi.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace DigraphyApi.Controllers;
+
+[Route("api/")]
```

Comment:
```suggestion
[Route("api/v1/[controller]")]
```

This will cause the route to be the class name, excluding the `Controller` suffix. You can then remove the `[Route]` from the endpoint. You'll need to rename the controller to be "Collection**s**Controller" though, with an 's'.